playbooks/rpm_packaging.yml

```diff
@@ -23,12 +23,18 @@
           - crudini
```

Comment:
This is no longer needed so I opened https://github.com/theforeman/foreman-packaging/pull/8545.

Also note that since https://github.com/theforeman/foreman-packaging/commit/37558e583e65819adb125942a3683785922004f0 click (on line 21) is no longer needed.